src/Services/SearchService.php

```diff
@@ -506,6 +517,26 @@ private function getVariationIdForRedirect(string $query, array $documents)
         return $cheapestVariationId;
     }
 
+    /**
+     * @return float|null
```

Comment:
How can this return `null`? Isn't the default price a `float`? `float` can be natively typed.